internal/cmd/generate.go

```diff
@@ -48,6 +54,23 @@ func (c *Generate) Execute(opts *commander.CommandHelper) {
 		os.Exit(1)
 	}
 
+	if !mustFollow {
+		code := generateCode(account)
+		fmt.Println(code)
+	} else {
+		previousCode := ""
+		for {
+			code := generateCode(account)
+			if code != previousCode {
+				fmt.Println(code)
+				previousCode = code
+			}
+			time.Sleep(time.Second)
+		}
+	}
```

Comment:
```suggestion
	if !mustFollow {
		code := generateCode(account)
		fmt.Println(code)

		return
	}
	
	previousCode := ""
	for {
		code := generateCode(account)
		if code != previousCode {
			fmt.Println(code)
			previousCode = code
		}
		time.Sleep(time.Second)
	}
```

Reply:
Or even remove the whole extra generate call:

```suggestion
	code := generateCode(account)
	fmt.Println(code)
	
	if !mustFollow {
		return
	}
	
	previousCode := code
	for {
		code := generateCode(account)
		if code != previousCode {
			fmt.Println(code)
			previousCode = code
		}
		time.Sleep(time.Second)
	}
```

Reply:
I've gone with the last suggestion. much cleaner indeed. Thanks!